src/main/java/wooteco/subway/line/dao/LineDao.java

```diff
@@ -82,30 +97,39 @@ private Line mapLine(List<Map<String, Object>> result) {
         List<Section> sections = extractSections(result);
 
         return new Line(
-                (Long) result.get(0).get("LINE_ID"),
-                (String) result.get(0).get("LINE_NAME"),
-                (String) result.get(0).get("LINE_COLOR"),
-                new Sections(sections));
+            (Long) result.get(0).get("LINE_ID"),
+            (String) result.get(0).get("LINE_NAME"),
+            (String) result.get(0).get("LINE_COLOR"),
+            (Integer) result.get(0).get("EXTRA_FARE"),
+            new Sections(sections)
+        );
     }
 
     private List<Section> extractSections(List<Map<String, Object>> result) {
         if (result.isEmpty() || result.get(0).get("SECTION_ID") == null) {
             return Collections.EMPTY_LIST;
```

Comment:
```suggestion
            return Collections.emptyList();
```

제네릭을 사용하는 메서드를 활용하면 어떠한 차이가 있을지 고민해보아요

Reply:
`Collections.EMPTY_LIST`는 제네릭이 아닌 `List`를 반환한다는 걸 처음 알았어요. 이 경우 타입 안전성이 저해될 수 있으므로, 제네릭을 사용하는 메서드로 수정했습니다!